tools/bridge/bridge/confirmation_sender.py

```diff
@@ -54,7 +58,15 @@ def run(self):
     def send_confirmation_transactions(self):
         while True:
             transfer_event = self.transfer_event_queue.get()
+
+            if not is_bridge_validator(self.home_bridge_contract, self.address):
```

Comment:
Could this check result in transfers getting stuck if lots of people start their node before they become validators? They see the transfer, ignore it (because they aren't a validator *at the moment*), and no one ever confirms it.

Reply:
Well this is a really good point. :scream: 
To be honest I fail to come up with an immediate solution. At least not for current setup. :thinking: 
Any suggestions from your side?

Reply:
In a meeting including @saschagoebel wie decided that this must not be solved within this PR. We do not come up with a perfect solution. Therefore I opened the issue #276. The problem is not critical, but should be solved soon.